src/websockets/server.py

```diff
@@ -714,7 +714,10 @@ async def _close(self) -> None:
         # asyncio.wait doesn't accept an empty first argument
         if self.websockets:
             await asyncio.wait(
-                [websocket.close(1001) for websocket in self.websockets],
+                [
+                    asyncio.create_task(websocket.close(1001))
```

Comment:
Note that `create_task()` was added in Python 3.7, and it looks like websockets will continue to support 3.6 in the next release: https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task